core/src/main/java/com/linecorp/armeria/common/HttpHeadersBase.java

```diff
@@ -174,11 +181,46 @@ URI uri() {
         }
     }
 
+    @Nullable
+    Locale selectLocale(Collection<Locale> supportedLocales) {
+        requireNonNull(supportedLocales, "supportedLocales");
+        final List<LanguageRange> languageRanges = acceptLanguages();
+        if (languageRanges == null) {
+            return null;
+        }
+        return languageRanges
+                .stream()
+                .flatMap(it -> Locale.filter(ImmutableList.of(it), supportedLocales).stream())
+                .findFirst()
+                .orElse(null);
+    }
+
+    @Nullable
+    List<LanguageRange> acceptLanguages() {
+        final List<String> acceptHeaders = getAll(HttpHeaderNames.ACCEPT_LANGUAGE);
+        if (acceptHeaders.isEmpty()) {
+            return null;
+        }
+
+        try {
+            final List<LanguageRange> acceptLanguages = new ArrayList<>(4);
+            for (String acceptHeader : acceptHeaders) {
+                acceptLanguages.addAll(LanguageRange.parse(acceptHeader));
+            }
+            acceptLanguages.sort(comparingDouble(LanguageRange::getWeight).reversed());
+
+            return ImmutableList.copyOf(acceptLanguages);
+        } catch (IllegalArgumentException e) {
+            // If any port of any of the headers is ill-formed
+            return null;
+        }
+    }
+
     HttpMethod method() {
         final String methodStr = get(HttpHeaderNames.METHOD);
         checkState(methodStr != null, ":method header does not exist.");
         return HttpMethod.isSupported(methodStr) ? HttpMethod.valueOf(methodStr)
-                                                 : HttpMethod.UNKNOWN;
+                : HttpMethod.UNKNOWN;
```

Comment:
nit: Revert?

Reply:
I haven't changed it in 34b738ec4c9182305e5df22121662e133a20dc82!? (latest pushed) Something wrong with Github?

Reply:
Maybe your IDE automatically formats the part unexpectedly.
I can see the diff [here](https://github.com/line/armeria/commit/34b738ec4c9182305e5df22121662e133a20dc82#diff-115d49fc03942185a7a36423eaf6f8cb36183533cd93909744ba8a4ba4b6e402R223).
Let me fix the indent. :-)